prow/config/config.go

```diff
@@ -43,6 +43,9 @@ type Config struct {
 	Plank  Plank  `json:"plank,omitempty"`
 	Sinker Sinker `json:"sinker,omitempty"`
 
+	// TODO: Move this out of the main config.
```

Comment:
Why?

Reply:
I expect that external agents that need their own fields, would wrap config.Config, extend it, and use that instead of making the change here.